README.md

```diff
@@ -1,20 +1,20 @@
-# Project Name
+# Odin-youtube-clone
 
-> One paragraph statement about the project.
+> This project is about embedding videos,images and using flexbox properties for positioning and also media query for responsiveness.
```

Comment:
This should be a full paragraph. I think it's too short. Can we describe what the project is about in detail? We can also add that it's a youtube player clone that demonstrates the knowledge of HTML iframe and images by embedding videos and images and flexbox was used for layout, you have a different CSS property for positioning.

Reply:
Yes, I got that,working on it immediately